kindelia_core/Cargo.toml

```diff
@@ -21,6 +21,7 @@ harness = false
 
 
 [dependencies]
+kindelia_ws_events = { path = "../kindelia_ws_events" }
```

Comment:
Also, it's suposed to be a dependency of `kindelia` node `kindelia_core`.

`kindelia` is the crate that will glue all the others communicating with `kindelia_core`.